pylint/checkers/raw_metrics.py

```diff
@@ -22,13 +22,13 @@
 
 
 def report_raw_stats(
-    sect,
+    sect: Section,
     stats: LinterStats,
     old_stats: LinterStats | None,
 ) -> None:
     """Calculate percentage of code / doc / comment / empty."""
     total_lines = stats.code_type_count["total"]
-    sect.description = f"{total_lines} lines have been analyzed"
+    sect.insert(0, Paragraph([Text(f"{total_lines} lines have been analyzed\n")]))
```

Comment:
Description needs to be passed to the constructor of a `Section`. This does nothing right now.

Reply:
Nice catch ! Apparently no one use this report 😅 ?